fixtures/nesting/src/legacy/createLegacyRoot.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import ThemeContext from './shared/ThemeContext';
+import {__RouterContext} from 'react-router';
+
+/* eslint-disable react/jsx-pascal-case */
+
+function Bridge({children, context}) {
+  return (
+    <ThemeContext.Provider value={context.theme}>
+      <__RouterContext.Provider value={context.router}>
```

Comment:
Context being bridged.

Reply:
Won't this potentially trigger the "context is used by more than one renderer concurrently" warning?

Reply:
I don't know (haven't seen that happen). How would that happen if we do this in the layout effect? Everything would be popped.

Reply:
At least in the fixture here we're using the same React version in /legacy and /modern. I tested with different versions and there's no warning.

Reply:
I'm confused about some of the comments about `react-router`.

Do you expect it to be duplicated or not? I.e. installed separately in `/modern` and separately in `/legacy`. Which might be two different versions of react-router.

If it's separately installed and duplicated then there's no problem here since it's two different context objects there are not two different renderers accessing it.

If it's not separately installed and instead uses the same instance of `react-router` then one of them has the wrong version of React when they created the context. So one of them will break. For example this call.

Reply:
>Do you expect it to be duplicated or not?

Yes.

>If it's separately installed and duplicated then there's no problem here since it's two different context objects there are not two different renderers accessing it. 

Ah. That explains why there was no warning, indeed.

>If it's not separately installed and instead uses the same instance of react-router then one of them has the wrong version of React when they created the context. 

Right. No, it's definitely duplicated.